src/internal/collection/postgres_collection.go

```diff
@@ -146,9 +146,9 @@ func (c *postgresCollection) ReadWrite(tx *pachsql.Tx) PostgresReadWriteCollecti
 
 // NewDryrunSQLTx is identical to NewSQLTx except it will always roll back the
 // transaction instead of committing it.
-func NewDryrunSQLTx(ctx context.Context, db *pachsql.DB, apply func(*pachsql.Tx) error) error {
+func NewDryrunSQLTx(ctx context.Context, db *pachsql.DB, apply func(context.Context, *pachsql.Tx) error) error {
```

Comment:
Smallest nit in the history of nits: `NewDryRunSQLTx`?

Reply:
I'm happy to rename it.  If you asked me what that function was called, I would have guessed your thing ;)

Reply:
Done.